apps/admin-gui/src/app/shared/components/dialogs/add-member-dialog/add-member-dialog.component.html

```diff
@@ -28,12 +28,12 @@ <h1 mat-dialog-title>{{'DIALOGS.ADD_MEMBERS.TITLE' | translate}}</h1>
         {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
       </button>
       <span
-        matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
-        matTooltipPosition="above"
-        [matTooltipDisabled]="selection.selected.length === 0 || inviteAuth">
+        matTooltip="{{(!showInvite && selection.selected.length > 0 ? 'DIALOGS.ADD_MEMBERS.INVITE_MEMBER_DISABLED' : 'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP') | translate}}"
+        [matTooltipPosition]="'below'"
+        [matTooltipDisabled]="(showInvite || selection.selected.length <= 0) || ((selection.selected.length === 0 || inviteAuth) && (showInvite || selection.selected.length <= 0))">
```

Comment:
I believe this could be simplified so it's easier to understand. Each clause checks if length of selected items is 0, which could be extracted.